schemas/system/Locking.category

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars, no-undef, handle-callback-err */
```

Comment:
Could you clarify the need for these?

If they were meant to handle the callback of `Lock` action then I'd prefer to just either put `() => {}` in there or comment out the arguments if they are needed for some reason.